.travis.yml

```diff
@@ -30,11 +30,13 @@ before_install:
 # command to install dependencies
 install:
   - pip install .
+  - pip install pylint
   - pip install yapf
```

Comment:
Should we remove yapf from the install/script portion, or is this not replacing yapf?

Reply:
Nope; I left a comment in the bug as well, but the TL;DR is that these tools do mostly different things (both of which are important).